library/core/benches/char/methods.rs

```diff
@@ -1,26 +1,26 @@
-use test::Bencher;
+use test::{black_box, Bencher};
 
 const CHARS: [char; 9] = ['0', 'x', '2', '5', 'A', 'f', '7', '8', '9'];
 const RADIX: [u32; 5] = [2, 8, 10, 16, 32];
 
 #[bench]
 fn bench_to_digit_radix_2(b: &mut Bencher) {
-    b.iter(|| CHARS.iter().cycle().take(10_000).map(|c| c.to_digit(2)).min())
+    b.iter(|| black_box(CHARS.iter().cycle().take(10_000).map(|c| black_box(c).to_digit(2)).min()))
```

Comment:
There is no need to put a `black_box` on the output, as it is automatically added by the `Bencher` interface: https://github.com/rust-lang/rust/blob/1.67.0/library/test/src/bench.rs#L117. The important part (for the output) is that the lambda function passed as argument to `.iter()` returns a non-`()` result (which is already the case here).

The `black_box` is definitely relevant for the input though :)